docs/docs.css

```diff
@@ -0,0 +1,170 @@
+/* font - quicksand */
+
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;600;700&display=swap');
+
+/* font - poppins  */
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+a{
+    text-decoration: none;
+}
+
+:root {
+    --navyBlue: #082032;
+    --waterBlue: #003566;
+    --offWhite: #F5F3FF;
+    --violet: #701a75;
+    --turquoise: #67e8f9;
+    --gray: #9ca3af;
+    --veryLightGray: #cbd5e1;
+    --charcoalBlack: #202020;
+    --lightBlack: #383838;
+    --zinc: #3f3f46; 
+}
+
+html{
+    background-color: var(--zinc);
+}
+
+body{
+    background-color: white;
+    font-family: 'Quicksand', sans-serif;
+    /* font-family: 'Poppins', sans-serif; */
+    margin: 0px;
+    height: 100vh;
```

Comment:
remove here also

Reply:
Implemented all the recomended changes in the commit #[c1c7865](https://github.com/mcmonojit/CoolWaterUI/pull/1/commits/c1c7865a9062a8223606e1d8ed9b9d4f9c688474)